docs/operations/tap.md

```diff
@@ -1,103 +1,289 @@
+    kubectl --namespace=storageos exec storageos-cli-77885d6d8b-zqmnn -- storageos describe volume pvc-abb18d51-7e1a-4812-8d65-40dbc090362a --namespace=default
+
+    ID                  882ab4dd-1b35-4bb4-a825-68763719b991
+    Name                pvc-abb18d51-7e1a-4812-8d65-40dbc090362a
+    Description
+    AttachedOn
+    Attachment Type     detached
+    NFS
+      Service Endpoint
+      Exports:
+    Namespace           default (5055ae9d-6278-4374-a6c8-e4779c6cc58f)
+    Labels              csi.storage.k8s.io/pv/name=pvc-abb18d51-7e1a-4812-8d65-40dbc090362a,
+                        csi.storage.k8s.io/pvc/name=pvc-tap,
+                        csi.storage.k8s.io/pvc/namespace=default,
+                        storageos.com/failure-mode=soft,
+                        storageos.com/nocompress=true,
+                        storageos.com/replicas=3,
+                        storageos.com/topology-aware=true,
+                        storageos.com/topology-key=custom-region
+    Filesystem          ext4
+    Size                5.0 GiB (5368709120 bytes)
+    Version             OQ
+    Created at          2022-07-22T16:05:04Z (42 minutes ago)
+    Updated at          2022-07-22T16:26:35Z (21 minutes ago)
+
+    Master:
+      ID                248ea74d-8753-4f64-afbf-73b72ddc211b
+      Node              demo-worker-node-4 (1c9284c7-99a4-40c5-9ab9-95df19c1a8ac)
+      Health            online
+      Topology Domain   1
+
+    Replicas:
+      ID                3172c40c-e745-48a9-91cf-39352389e99e
+      Node              demo-worker-node-5 (37fdc95a-e215-44e7-a53d-45c1b7a7bad1)
+      Health            ready
+      Promotable        true
+      Topology Domain   2
+
+      ID                516bb457-4720-4b65-af70-327fb7d74898
+      Node              demo-worker-node-3 (669e2d13-2520-4238-9be6-4f58540f0f64)
+      Health            ready
+      Promotable        true
+      Topology Domain   3
+
+      ID                e48d6084-8ce8-4d57-8644-20d61c28005e
+      Node              demo-worker-node-1 (114ae6a7-c40d-40c2-87cb-1dc9dcc24348)
+      Health            ready
+      Promotable        true
+      Topology Domain   1
+    ```
+
+    > 💡  As demonstrated in the output above, notice how the master volume and each replica volume are deployed on a different nodes (`demo-worker-node-4`, `demo-worker-node-5`, `demo-worker-node-3` and `demo-worker-node-1` respectively) to ensure data protection and high availability in the event of a transient node failure.
+
+### Example - Enable Topology-Aware Placement Through a `StorageClass` Definition
+
+The following guidance below will demonstrates how to use Ondat Topology-Aware Placement through a `StorageClass` definition. 
+- The instructions will enable Topology-Aware Placement through a custom StorageClass, use the node label >> `topology.kubernetes.io/zone` and set the default volume replica count >> `storageos.com/replicas` to `2`.
```

Comment:
is the `>>` intended?

Reply:
yes - I was use it point to specific items, but I am happy to remove it